backend/controllers/prisonerProfile/prisonerSentenceAndRelease.js

```diff
@@ -28,15 +28,20 @@ module.exports = ({ prisonerProfileService, prisonApi, systemOauthClient }) => a
   const sentenceAdjustments = sentenceAdjustmentsViewModel(sentenceAdjustmentsData)
   const courtCases = courtCasesViewModel({ courtCaseData, sentenceTermsData, offenceHistory })
 
+  const determineLifeSentence = async () => {
+    const prisonerDetails = await prisonApi.getPrisonerDetails(res.locals, offenderNo)
```

Comment:
This call to `getPrisonerDetails` returns is a little bit heavy and returns a lot of info - so kept it separate and it only runs in a specific circumstance when no release date is defined (so should run rarely)

One for discussion with the dev team - we could potentially improve performance by using the ElasticSearch `prisonerSearchService` rather than the `prisonApi`

Reply:
Yes, this is covered by a ticket which is currently in Initial Elab: https://dsdmoj.atlassian.net/browse/NN-3272